geode-assembly/build.gradle

```diff
@@ -193,19 +195,20 @@ def cp = {
         it.contains('log4j-slf4j-impl') ||
         it.contains('shiro') ||
         it.contains('slf4j-api') ||
+        it.contains('spring-beans') ||
```

Comment:
This might cause a conflict with jars on the `gemfire-web` classpath.

Reply:
The reason this was added is that `CommandManager` will now try to load any classes implementing `CommandMarker`, rather than only those in the `org.apache.geode.management.internal.cli.commands` package.  This results in the CommandManager trying recognizing the built-in `HelpCommands` command from spring-shell, and that class has a dependency on spring-beans.  We could probably exclude classes by name rather than by an actual reference to the `Class` to avoid loading blacklisted commands (and thus remove the need for this dependency).